deploy/kubernetes/overlays/stable/ecr/kustomization.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
-- ../../../base
+  - ../../../base
 images:
-- name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
-  newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver
-  newTag: v0.9.0
-- name: k8s.gcr.io/sig-storage/csi-provisioner
-  newName: public.ecr.aws/eks-distro/kubernetes-csi/external-provisioner
-  newTag: v2.0.3-eks-1-18-1
-- name: k8s.gcr.io/sig-storage/csi-attacher
-  newName: public.ecr.aws/eks-distro/kubernetes-csi/external-attacher
-  newTag: v3.0.1-eks-1-18-1
-- name: k8s.gcr.io/sig-storage/livenessprobe
-  newName: public.ecr.aws/eks-distro/kubernetes-csi/livenessprobe
-  newTag: v2.1.0-eks-1-18-1
-- name: k8s.gcr.io/sig-storage/csi-node-driver-registrar
-  newName: public.ecr.aws/eks-distro/kubernetes-csi/node-driver-registrar
-  newTag: v2.0.1-eks-1-18-1
+  - name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
+    newName: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver
+    newTag: v0.9.0
```

Comment:
ecr build of v0.10.x is still pending.

once built, need to bump this in master and in release-0.10 branch.